Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -169,6 +172,10 @@ - (void)resumePreviousSessionReplay:(SentryEvent *)event
     eventContext[@"replay"] =
         [NSDictionary dictionaryWithObjectsAndKeys:replayId.sentryIdString, @"replay_id", nil];
     event.context = eventContext;
+
+    if ([NSFileManager.defaultManager removeItemAtURL:lastReplayURL error:&error] == NO) {
+        SENTRY_LOG_ERROR(@"Can`t delete '%@': %@", SENTRY_LAST_REPLAY, error);
+    }
```

Comment:
We should expose `SentryFilemanager._non_thread_safe_removeFileAtPath` and reuse it here, it has even better logging for different kinds of errors

Reply:
Thanks for the tip, It makes sense, I will create a note for refactoring later.